pessimistic-concurrency/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>
+            java-design-patterns
+        </artifactId>
+        <groupId>
+            com.iluwatar
+        </groupId>
+        <version>
+            1.26.0-SNAPSHOT
+        </version>
+    </parent>
+    <modelVersion>
+        4.0.0
+    </modelVersion>
+
+    <artifactId>
+        pessimistic-concurrency
+    </artifactId>
+
+    <properties>
+        <maven.compiler.source>
+            17
+        </maven.compiler.source>
+        <maven.compiler.target>
+            17
+        </maven.compiler.target>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.hibernate.javax.persistence</groupId>
+            <artifactId>hibernate-jpa-2.1-api</artifactId>
+            <version>1.0.2.Final</version>
```

Comment:
Use the dependencies specified in the parent pom.xml and leave out the versions here